test/spec/exception-handling/binary.txt

```diff
@@ -0,0 +1,280 @@
+;;; TOOL: run-interp-spec
```

Comment:
Does this test ([`binary.wast`](https://github.com/WebAssembly/testsuite/blob/main/proposals/exception-handling/binary.wast) in testsuite) have any EH related features? Can't find any. Also no modules seem to contain the tag section.

Reply:
It does, though it's subtle. The upstream spec tests encode some error cases that conflict with exceptions.

For example, it checks for an invalid section id here: https://github.com/WebAssembly/spec/blob/main/test/core/binary.wast#L48

That is updated in the exception tests: https://github.com/WebAssembly/exception-handling/blob/main/test/core/binary.wast#L48

(because the old invalid section id 13 is now a valid tag section)